client_plugin/Makefile

```diff
@@ -361,11 +361,11 @@ test-e2e:
 .PHONY: test-e2e-runalways test-e2e-runonce
 test-e2e-runalways:
 	$(log_target)
-	$(GO) test -v -timeout 30m -tags runalways $(E2E_Tests)
+	$(GO) test -v -timeout 40m -tags runalways $(E2E_Tests)
```

Comment:
Unfortunately to see this happening :) Do we have an issue to track the improvement of total testing time?